chips/veer_el2/src/chip.rs

```diff
@@ -288,11 +242,11 @@ pub unsafe extern "C" fn start_trap_rust() {
 /// mcause is passed in, and this function should correctly handle disabling the
 /// interrupt that fired so that it does not trigger again.
 #[export_name = "_disable_interrupt_trap_rust_from_app"]
-pub unsafe extern "C" fn disable_interrupt_trap_handler(mcause_val: u32) {
```

Comment:
I think the function should remain unsafe

Reply:
I've fixed this.